cc-env_test.go

```diff
@@ -68,6 +68,7 @@ func makeRuntimeConfig(prefixDir string) (configFile string, config oci.RuntimeC
 	shimPath := filepath.Join(prefixDir, "cc-shim")
 	proxyPath := filepath.Join(prefixDir, "cc-proxy")
 	disableBlock := true
+	blockStorageDriver := "virtio-scsi"
```

Comment:
Seeing this change makes me wonder if we should be displaying it in the ` [Hypervisor]` section of the `cc-runtime cc-env` output?

Reply:
Makes sense to show this in the cc-env output. I have included this in the PR now.

Reply:
Thanks! Could you bump `formatVersion` to `1.0.9` as this is a `cc-env` format change.